src/components/header.scss

```diff
@@ -4,7 +4,7 @@
   position: fixed;
   display: flex;
   justify-content: center;
-  z-index: 10;
+  z-index: 100;
```

Comment:
Is this necessary? Not sure if you are addressing something in this section or a bug elsewhere.

Reply:
I must have put that in a little early, I was having an issue with something overlapping the header. I'm not able to find it anymore though.